pytrip/vdx.py

```diff
@@ -1287,6 +1278,26 @@ def get_min_max(self):
         self.temp_max = temp_max
         return temp_min, temp_max
 
+    def is_fully_contained(self):
```

Comment:
Add docstring explaining what this method is doing.
Take some inspiration from 
https://github.com/pytrip/pytrip/pull/589/files#diff-3db93b3f0fd4220f8fa3abdfe45db89baae38ab57a33c3109f2eaab2f3c5eccfR809